extension/services/youtube/youtube.js

```diff
@@ -90,6 +91,36 @@ class YouTube extends serviceList.Service {
       await this.callOneTab(tab.id, "move", { direction });
     }
   }
+
+  async playAlbum(query) {
```

Comment:
Could this just be:

```js
async playAlbum(query) {
  return this.playQuery(query + " album");
}
```

?

Reply:
Yes, I was thinking it at first but giving query to `playQuery(query + " album");` has inconsistencies like if we give query like " purpose album ", then it would be converted to "purpose album youtube.com" then it will do an I'm feeling lucky google search in which it will not show album playlist and just play a song from the album.

Reply:
![Screen-Recording-2020-03-28-at-9](https://user-images.githubusercontent.com/22199666/77814408-d5c52200-70d6-11ea-877d-2acd822904cf.gif)
This happens when we search for " purpose album " on youtube now. But this would require first going to the youtube website itself.